crates/ruff_linter/src/checkers/ast/analyze/expression.rs

```diff
@@ -776,6 +776,9 @@ pub(crate) fn expression(expr: &Expr, checker: &mut Checker) {
             if checker.enabled(Rule::NestedMinMax) {
                 pylint::rules::nested_min_max(checker, expr, func, args, keywords);
             }
+            if checker.enabled(Rule::RepeatedKeywords) {
+                pylint::rules::repeated_keywords(checker, keywords);
```

Comment:
Most rules don't yet follow this pattern, but we tend to prefer passing `call` here in lieu of `keywords`, and then accessing `call.keywords` as-needed within `repeated_keywords`. The benefit is that we ensure callers pass an AST node of the right type, rather than passing some arbitrary vector of keywords. That benefit isn't so clear in _this_ case, but consider the call to `nested_min_max` right above, where the third argument is just an arbitrary `&Expr`. A caller could pass _any_ expression, but in truth, we'd prefer that call to take an `ExprCall`.